blazor/datagrid/data-binding.md

```diff
@@ -1198,6 +1198,95 @@ The following sample code demonstrates notifying user when server-side exception
     }
 }
 ```
+
+## Perform data operation in WebAPI service
+
+While using the WebAPI adaptor type in the SfDataManager component, queries will be generated for data operations to be sent to the server side as query strings, and based on these query string values, actions(filtering, sorting, paging, etc.) will be performed on the server side.
+
+Requests sent to the server can be retrieved using "Request.Query", and these values can be differentiated as below. create a table or points using $skip, $top, $filter, $search , $sort along with the explanation.
```

Comment:
create a table is for your reference. create a table with key and explanation. and explain these words